packages/create/src/generators/app/header.js

```diff
@@ -0,0 +1,16 @@
+import chalk from 'chalk';
+
+export default `
+        _.,,,,,,,,,._
+     .d''           \`\`b.       ${chalk.underline('Open Web Components Recommendations')}
+   .p'      Open       \`q.
+ .d'    Web Components  \`b.    Start or upgrade your web component project with
+ .d'                     \`b.   ease. All the nice setting at your fingertips.
+ ::   .................   ::
+ \`p.                     .q'   See more details at https://open-wc.org/create/
+  \`p.    open-wc.org    .q'
+   \`b.     @openWc     .d'
+     \`q..            ..,'      Note: you can exit any time with Strg+C or Esc
```

Comment:
```suggestion
     \`q..            ..,'      Note: you can exit any time with Ctrl+C or Esc
```

(is strg german? :p)

Reply:
jup 🙈